packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -280,7 +281,7 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         if (!this.channelDeferred) {
             this.channelDeferred = new Deferred<IFluidDataStoreChannel>();
             this.realizeCore().catch((error) => {
-                this.channelDeferred?.reject(error);
+                this.channelDeferred?.reject(CreateProcessingError(error, undefined));
```

Comment:
Do you also want to do the same in the catch block inside `bindRuntime`?

Reply:
i don't think so. i'm really just looking to cover cases where we are processing the snapshot. bind runtime does do some op processing, but that will be caught in the channel handler which was added previously to cover op processing